core/kits/documents/tabs/tab-base.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
Instead of duplicating a bunch of methods, why not call `$this->document->add_control`?

Reply:
Because our standard is to call only `$this->add_control` in order to easier move controls between classes. this class will be a base also for Skin_Base.

Reply:
If so, this class has to be located in another place